app/src/main/java/com/github/se/polyfit/ui/components/scaffold/BottomNavigation.kt

```diff
@@ -61,11 +68,16 @@ fun BottomNavigationBar(
         label = {
           if (currentRoute == Route.PostInfo)
               Text(
-                  context.getString(R.string.map_nav_label),
+                  if (showingMap) context.getString(R.string.map_nav_label)
+                  else context.getString(R.string.map_nav_posts),
```

Comment:
Using a switch might be good or even something like
```
text = context.getString(R.string.map_nav_label).takeIf{showingMap} ?: context.getString(R.string.map_nav_posts)
```
This works similarly to an if but doesn't add to the cognitive complexity

Reply:
Yeah, you're right. I need to train my brain to think in cognitive complexity. Thanks for the tip.

Reply:
you're welcome, I just learned about that myself